tools/macos/README.md

```diff
@@ -7,7 +7,7 @@ One way to develop or deploy OpenWhisk on a Mac is to use [docker-machine](https
 The following are required to build and deploy OpenWhisk from a Mac host:
 
 - [Oracle VM VirtualBox](https://www.virtualbox.org/wiki/Downloads)
-- [Docker 1.9.1](https://docs.docker.com/engine/installation/mac/) (including `docker-machine`)
+- [Docker 1.12.0](https://docs.docker.com/engine/installation/mac/) (including `docker-machine`)
```

Comment:
Sure but can you pin only minor versions in brew and docker-machine? We need to provide valid URLs after all. No problem with apt tho.